pegasus/sites.v3/code.org/public/blockchain.haml

```diff
@@ -0,0 +1,120 @@
+---
+title: How Blockchain Works
+theme: responsive_full_width
+---
+
+%link{href: "shared/css/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/blockchain-styles.css", rel: "stylesheet"}
+
+%section.hero-banner-basic.bg-neutral-dark{style: "display: block !important"}
+  .wrapper
+    .text-wrapper.col-60
+      %h1{style: "color: white"}=hoc_s(:blockchain_hero_heading)
+      %p{style: "color: white"}=hoc_s(:blockchain_hero_desc)
+    %figure.col-33{style: "float: right"}
+      %img{src: "/images/banners/blockchain-laptop-silver.png", alt: "", style: "position: relative; width: 100%;"}
+    .clear
+
+%section
+  .wrapper
+    %h2=hoc_s(:blockchain_intro_heading)
+    %p=hoc_s(:blockchain_intro_desc)
+    %div.col-50{style: "margin-bottom: 1em"}
+      %p
+        %strong=hoc_s(:blockchain_intro_list_heading)
+      %ul
+        %li=hoc_s(:blockchain_intro_list_item_01)
+        %li=hoc_s(:blockchain_intro_list_item_02)
+        %li=hoc_s(:blockchain_intro_list_item_03)
+    %figure.col-45{style: "float:right; margin: 2em auto;"}
+      %img{src: "/images/blockchain-works-logo.png", alt: "", style: "width: 100%"}
+    .clear
+    .video-series
+      %figure.video-responsive{style: "max-width: unset"}
+        %div
+          %iframe{src: "https://www.youtube-nocookie.com/embed/MjibPL4tGqo", allowFullScreen: "true", frameborder: "0"}
+      %figure.video-responsive{style: "max-width: unset"}
+        %div
+          %iframe{src: "https://www.youtube-nocookie.com/embed/10wYLCnDHao", allowFullScreen: "true", frameborder: "0"}
+      %figure.video-responsive{style: "max-width: unset"}
+        %div
+          %iframe{src: "https://www.youtube-nocookie.com/embed/It_Ivgm5wQA", allowFullScreen: "true", frameborder: "0"}
+      %figure.video-responsive{style: "max-width: unset"}
+        %div
+          %iframe{src: "https://www.youtube-nocookie.com/embed/3EwkfUz_BPs", allowFullScreen: "true", frameborder: "0"}
+      %figure.video-responsive{style: "max-width: unset"}
+        %div
+          %iframe{src: "https://www.youtube-nocookie.com/embed/d9rKkk3oVdY", allowFullScreen: "true", frameborder: "0"}
+      %figure.video-responsive{style: "max-width: unset"}
+        %div
+          %iframe{src: "https://www.youtube-nocookie.com/embed/WzxbwEQRJPo", allowFullScreen: "true", frameborder: "0"}
+      %figure.video-responsive{style: "max-width: unset"}
+        %div
+          %iframe{src: "https://www.youtube-nocookie.com/embed/C3C_JfG8cNg", allowFullScreen: "true", frameborder: "0"}
+
+= view :section_divider_line
+
+%section
+  .wrapper
+    %h2=hoc_s(:blockchain_lessons_heading)
+    %p=hoc_s(:blockchain_lessons_desc)
+    .action-block__wrapper.action-block__wrapper--three-col
+      .action-block.action-block--one-col
+        %p.overline Grade Levels: 7-12
+        %h3 Why Blockchain
```

Comment:
Do these headers need to be localized as well? It looks like there are 4 h3 tags here that aren't

Reply:
Ah thanks for asking! I talked to the intl team and we're just going to translate the header and description in that section and append `(English only)` to the translated strings.

Here's the [Slack thread](https://codedotorg.slack.com/archives/C04P9NMPNSD/p1678744900484549?thread_ts=1678470944.259829&cid=C04P9NMPNSD) if you want to read more (I'll also link to it above).

Reply:
Ah, I see! Sorry, I missed your note about that –– sounds good!

Reply:
😌